### ISSUE TITLE
filePath("/..", removeUps = TRUE) should give an error

### ISSUE THREAD

**Issue by HenrikBengtsson** [HenrikBengtsson/R.utils#141]
The path `/..` cannot exist, but we still get:

```r
> R.utils:::filePath("/..", removeUps = TRUE)
[1] ""
```

This should be an error, just like:

```r
> R.utils:::filePath("/../..", removeUps = TRUE)
Error in removeUpsFromPathname(pathname) : 
  Non-existing, or invalid, path: '/../..'
```

Found while troubleshooting #119

**Comment by HenrikBengtsson** [comment]
Fixed in commit 563c507a9. However, by design, `filePath()` should not give an error here.  Instead, it'll return the maximum pruned path.

**Comment by HenrikBengtsson** [comment]
However, according to the POSIX standards, the parent of the root directory is the root directory, cf. https://superuser.com/questions/186184/the-parent-of-the-root-directory.

So, ideally we should return `/` for `/..`,  `/../..`,  `/../../..`, and so on.
